b2sdk/file_version.py

```diff
@@ -84,7 +87,7 @@ def as_dict(self):
             'fileId': self.id_,
             'fileName': self.file_name,
             'fileInfo': self.file_info,
-            'legalHold': self.legal_hold,
+            'legalHold': bool(self.legal_hold) if self.legal_hold is not None else None,
```

Comment:
if unknown -> ValueError

Reply:
nooooo! A user can easily get an unknown legal_hold (because they are not authorized to read it) and this method is used by CLI to display contents of LS and such. So we definitely need to display an unknown value here.

Reply:
:D :P